_posts/2016-08-25-hybox-names.markdown

```diff
@@ -0,0 +1,38 @@
+
+As we’ve moved through the first year of the project, it has become clear that our effort has many facets, and choosing (and using) the right names has helped differentiate the various strands. We have identified at least five distinct components for our effort, each with its own identity and branding considerations.
+
+1. **the project.** We’ve settled on using **Hydra-in-a-Box** or **HyBox** (for shorthand, and used with some reservation) for the project. The project URL is currently [hydrainabox.org](http://hydrainabox.org), and resolves to [http://hydrainabox.projecthydra.org/](http://hydrainabox.projecthydra.org/). While not all of the work revolves around Hydra (there are significant tie-ins specific to DPLA hubs and DPLA metadata--see below), this name captures one of the essential goals of the effort: to produce a turnkey repository product leveraging, and embraced by, the Hydra community. 
+
+2. **code components.** Hydra’s technical architecture has many components and layers, and an explicit Hydra-in-a-Box goal is to contribute to the overall architecture and code ecosystem. Note that a site may choose to run these HyBox components but *not *run HyBox in its turnkey form, just as many sites now say they run "sufia-based repositories." Under the theory that a rising tide raises all ships, the project team’s focus has been to locate functionality in the right places within the Hydra stack, and *not *to create a product-specific code base where we can avoid it. Accordingly, we are **not** using HyBox or Hydra-in-a-Box as the name of code components, in order to encourage re-use in other projects, broad adoption, and code contribution & maintenance from other Hydra efforts.
+
+Therefore, for new code produced by the Hydra-in-a-Box effort, we have first sought to locate it where it rightfully belongs in the stack (e.g., Sufia, an independent gem, or even Fedora). Our hope is that the work we’re doing is general and modular enough that other Hydra efforts can and will make use of our code components--e.g., it would be excellent for [the Avalon Media System](http://www.avalonmediasystem.org/) to make use of (and contribute back to!) our components when it needs similar functionality. 
+
+3. **turnkey repository product,** locally managed. Building on what the Hydra community has learned working on Sufia to make it easy to install and maintain, we expect to provide a complete, configurable, turnkey product--more akin to Avalon, DSpace, or even ContentDM. We need a name for this product analogous to those systems, and with broad market appeal and strong branding identity. We have put a lot of thought into requirements for a good name: 
```

Comment:
I think the first sentence can be tightened up a bit. (Yeah, I know, I wrote part of it. ;) ) How about changing "has learned working to make Sufia easy to install and maintain?" That obviates the need for the "it" there, and one fewer pronoun to resolve is a good thing IMO.